docs/389ds/design/audit-json-logging-design.md

```diff
@@ -0,0 +1,73 @@
+---
+title: "Audit JSON Logging Design"
+---
+
+# Audit JSON Logging
+----------------
+
+Overview
+--------
+
+In order to make the audit logging more consumable by standard parsing tool it will written in a JSON format instead of using a partial LDIF format which not LDAP compliant in its current form.
+
+While the old format, after a bit if data massaging, could be replayed using a LDAP client (e.g. ldapmodify), this is most likely not used by most customers.  Instead, customers are looking for actual audit information that can by consumed by well known log parsers.
+
+JSON Design
+------------
+
+```
+[
+    {
+        date: 20243457458475847Z
+        target_dn: DN,
+        bind_dn: DN,
+        client: IP_ADDRESS,
+        conn_id: ####,
+        op_id: ####,
+        changetype: add/delete/modify/modrdn,
```

Comment:
I think that today we also have the result

Reply:
haha, I really thought I already had that in there.  Nice catch, I'll add it now...